mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -138,7 +143,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
         showProgressDialog(Constants.PLEASE_WAIT);
         mEditProfilePresenter.fetchUserDetails();
-
+        Bitmap bitmap = load();
```

Comment:
There are multiple problems with this line.
1. The File is being read on the main thread.
2. The File is being read inside the activity which is wrong because business logic needs to live in the `Presenter`.